docs/_assets/style.css

```diff
@@ -12,12 +12,37 @@ mdjs-preview {
   margin-bottom: 20px;
 }
 
-#main-header a[href='/blog/'] {
-  display: none;
+#main-header a {
+  font-weight: normal;
 }
 
-@media screen and (min-width: 1024px) {
+#main-header a.current.active {
+  font-weight: bold;
```

Comment:
You should keep them bold even when it is not the `current` page, as in when you are on a sub-page.